guides/common/modules/proc_synchronizing-content-over-a-network.adoc

```diff
@@ -0,0 +1,85 @@
+
+.Procedure
+. Navigate to *Content* > *Subscriptions*.
+. Click the *Manage Manifest* button.
+. Navigate to the *CDN Configuration* tab.
+. Select the *Upstream {ProjectServer}* tab.
+. In the *URL* field, enter the address of the upstream {ProjectServer}.
+. In the *Username*, enter your username for upstream login.
+. In the *Password*, enter your password or personal access token for upstream login.
+. In the *Organization label* field, enter the label of the upstream organization.
+. Optional: In the *Lifecycle Environment Label* field, enter the label of the upstream lifecycle environment.
+Default is `Library`.
+. Optional: In the *Content view label* field, enter the label of the upstream Content View.
+Default is `Default_Organization_View`.
+. From the *SSL CA Content Credential* menu, select a CA certificate used by the upstream {ProjectServer}.
+. Click *Update*.
+. Navigate to *Content* > *Products*.
+. Select repositories to synchronize.
+. Click *Sync Now*.
+
+You can also create a synchronization plan to ensure updates on a regular basis. For more information, see xref:Creating_a_Sync_Plan_{context}[].
+
+.CLI Procedure
+. Show some related info under the "CDN Configuration" section in the output // TODO
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer organization show --id=1
+----
+. Update CDN config to an upstream server
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer organization configure-cdn --id=11 --type=network_sync \
+--username _upstream_username_ --password _upstream_password_ \
+--ssl-ca-credential-id 8 \
+--upstream-organization-label="_My_Organization_" \
+[--upstream-lifecycle-environment-label="_My_Lifecycle_Environment_"] \
+[--upstream-content-view-label="_My_Content_View_"]
+----
+
+////
+
+. Update cdn config to Red Hat CDN:
+# hammer -u admin -p changeme organization configure-cdn --id=11 --type=redhat_cdn
+
+. Update to airgapped mode:
+# hammer -u admin -p changeme organization configure-cdn --id=11 --type=export_sync
+
+It occurs to me that the type values of upstream_server and airgapped should be changed to network_sync and export_sync respectively. I'll file a bug for it
+
+Usage:
+    hammer organization configure-cdn [OPTIONS]
+
+Options:
+ --id VALUE                                   Id of the Organization
+ --label VALUE                                Organization label to search by
+ --name VALUE                                 Organization name to search by
+
+ --password, -p VALUE                             Password for authentication. Relevant only for ‘upstream_server’ type.
+ --ssl-ca-credential-id NUMBER                Content Credential to use for SSL CA. Relevant only for ‘upstream_server’ type.
+ --title VALUE                                Organization title
+ --type VALUE                                 CDN configuration type. One of redhat_cdn, upstream_server, airgapped.
```

Comment:
These options are now `redhat_cdn`, `network_sync` and `export_sync`.

`upstream_server` was renamed to `network_sync`
`airgapped` was renamed to `export_sync`

Reply:
This is not supposed to be here + see above.